### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by keeper335**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
     "type": "git",
     "url": "git+https://github.com/keeper335/node_gmp_homework1.git"
   },
+  "dependencies": {
+    "node-csvtojson": "git://github.com/Keyang/node-csvtojson"
```
Comment (Ivanko5417) [inline package.json]: it's better to use packages from npm. if the repository is deleted, you won't be able to use this pacakge

**Review by Ivanko5417 — CHANGES_REQUESTED** [review]
Looks like you haven't implemented task 1.1

**Comment by keeper335** [comment]
> Looks like you haven't implemented task 1.1
It's implemented in the first commit, unfortunately can't push the initial commit into PR otherwise need to create new repo

File: package.json (hunk 1 above)
Comment (keeper335) [reply package.json]: This is just a way to comply task requirements to use the mentioned link as is

**Comment by Ivanko5417** [comment]
could you please specify the link to the commit? I've checked all of them and haven't found the 1.1 task

**Review by Ivanko5417 — CHANGES_REQUESTED** [review]
You can create one more commit with the 1.1 Task

**Comment by keeper335** [comment]
> could you please specify the link to the commit? I've checked all of them and haven't found the 1.1 task

https://github.com/keeper335/node_gmp_homework1/commit/6e98ef1ef65b46fb9d9e9aa30dfd1364fce36fc6

**Review by Ivanko5417 — APPROVED** [review]

**Pull request merged**